ember-cli/2017-06/june-15.md

```diff
@@ -0,0 +1,10 @@
+# ember-cli Meeting Notes - 2017-06-15
+
+## Attendees
+
+
+## Agenda
+
```

Comment:
* Testing
  * deprecate usage of restricted resolver - https://github.com/emberjs/rfcs/pull/229
  * new testing API - https://github.com/emberjs/rfcs/pull/232